GitCommands/AppTitleGenerator.cs

```diff
@@ -46,10 +50,17 @@ public string Generate(string workingDir = null, bool isValidWorkingDir = false,
             var description = _description.Get(workingDir);
 
 #if DEBUG
-            return $"{description} ({branchName}) - Git Extensions [DEBUG]";
+            return $"{description} ({branchName}) - Git Extensions [{_buildBranch}]";
 #else
             return $"{description} ({branchName}) - Git Extensions";
+#endif
+        }
+
+        public static void Initialise(string buildBranch)
+        {
+#if DEBUG
+            _buildBranch = buildBranch;
```

Comment:
Turning this into a property would improve readability.

Reply:
done. I followed the pattern in UserEnvironmentInformation. (no static field either)